### PR TITLE
Replace anchor.js with jekyll-anchor-headings.

### PR DESCRIPTION
**Pull request by XhmikosR**
https://github.com/allejo/jekyll-anchor-headings

Shout-out to @allejo <3

### CODE REVIEW

**Comment by XhmikosR** [comment]
@mdo: thoughts?

Either way is fine by me. This solution gets rid of the JS dep, but it's small anyway.

File: site/_includes/anchor_headings.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        <!-- The placement of our anchor, before the heading content or after -->
+        {% if beforeHeading %}
+            {% capture _current %}<h{{ _hAttrToStrip | raw }}{{ anchor }}{% endcapture %}
+            {% capture edited_headings %}{{ edited_headings }}{{ node | replace: _hAttrToStrip, _current | raw }}{% endcapture %}
+        {% else %}
+            {% capture _current %}<h{{ _workspace | first }}{{ anchor }}</h{{ _workspace | last }}{% endcapture %}
+            {% capture edited_headings %}{{ edited_headings }}{{ _current }}{% endcapture %}
+        {% endif %}
+    {% endfor %}
+{% endcapture %}{% assign headingsWorkspace = '' %}{{ edited_headings | strip }}
```
Comment (MartijnCuppens) [inline site/_includes/anchor_headings.html]: Maybe add a newline after at the end of file?

Comment (MartijnCuppens) [reply site/_includes/anchor_headings.html]: This file is also indented with 4 spaces

File: site/docs/4.1/assets/js/src/application.js [hunk 2]
```diff
@@ -101,10 +101,6 @@
         .tooltip('_fixTitle')
     })
 
-    anchors.options = {
-      icon: '#'
-    }
-    anchors.add('.bd-content > h2, .bd-content > h3, .bd-content > h4, .bd-content > h5')
     $('.bd-content > h2, .bd-content > h3, .bd-content > h4, .bd-content > h5').wrapInner('<div></div>')
```
[the remark this answers is not in the document]
Comment (MartijnCuppens) [reply site/docs/4.1/assets/js/src/application.js]: We can perfectly move this to `anchor_headings.html`

[the remark this answers is not in the document]
Comment (MartijnCuppens) [reply site/docs/4.1/assets/js/src/application.js]: And clean up this a bit:
https://github.com/twbs/bootstrap/blob/53d279b934f8b53df5c62b504848579ccc7c8bcb/site/docs/4.1/assets/scss/_content.scss#L10-L19

**Comment by XhmikosR** [comment]
Like I said its third party file so I'd rather not touch it.

**Comment by MartijnCuppens** [comment]
Maybe move it to a vendor folder in the `_includes` than?

**Comment by XhmikosR** [comment]
I don't know, I didn't like adding yet another subfoler in the includes. I will think about it, but it's not something that blocks this.

File: site/docs/4.1/assets/js/src/application.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (XhmikosR) [reply site/docs/4.1/assets/js/src/application.js]: You can push to this branch if you want.

**Comment by XhmikosR** [comment]
OK, moved the include to a vendor subfolder.

**Comment by XhmikosR** [comment]
@MartijnCuppens: should we ask upstream for a custom tag wrapping support?

File: site/docs/4.1/assets/scss/_anchor.scss [hunk 3]
```diff
@@ -1,10 +1,15 @@
-.anchorjs-link {
+.bd-anchor {
+  padding-left: .25rem;
   font-weight: 400;
-  color: rgba($link-color, .5);
-  @include transition(color .16s linear);
+  color: $link-color;
+  opacity: 0;
```
Comment (XhmikosR) [inline site/docs/4.1/assets/scss/_anchor.scss]: Can someone verify if this needs `visibility: hidden` too?

Comment (MartijnCuppens) [reply site/docs/4.1/assets/scss/_anchor.scss]: It doesn't. You can't transition visibility: hidden, but you can transition the opacity.

**Comment by MartijnCuppens** [comment]
Yeah, why not. I've also made a PR to indent the file with two spaces here: https://github.com/allejo/jekyll-anchor-headings/pull/4

**Review by MartijnCuppens — APPROVED** [review]

**Pull request merged**